awx/ui_next/src/screens/Host/HostDetail/HostDetail.jsx

```diff
@@ -18,13 +23,114 @@ const ActionButtonWrapper = styled.div`
   }
 `;
 
-function HostDetail({ host, i18n }) {
+function HostDetail({
+  host,
+  history,
+  isDeleteModalOpen,
+  match,
+  i18n,
+  toggleError,
+  toggleLoading,
+  onHostDelete,
+  onToggleDeleteModal,
+  onToggleError,
+  onHandleHostToggle,
+}) {
   const { created, description, id, modified, name, summary_fields } = host;
+  let createdBy = '';
+  if (created) {
+    if (summary_fields.created_by && summary_fields.created_by.username) {
+      createdBy = (
```

Comment:
We could use the `< UserDateDetail />` component for the "created and modified by" details.